src/System.Console/tests/project.lock.json

```diff
@@ -424,11 +406,11 @@
           "lib/DNXCore50/System.Threading.ThreadPool.dll": {}
         }
       },
-      "xunit/2.1.0": {
+      "xunit/2.1.0-beta3-build3029": {
```

Comment:
Can you please rebase your changes on top of master and see if that fixes the issue?  We worked through all of this stuff on last Friday.  These lock file updates are going to cause problems if we merge them as is.